content/docs/rfcs/10/README.md

```diff
@@ -76,13 +79,15 @@ For example:
 For more on the concept of adaptive nodes and what this means in practice,
 please see the [30/ADAPTIVE-NODES](/spec/30) spec.
 
-## Network interaction domains
+## Network Interaction Domains
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “NOT RECOMMENDED”, 
+“MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-While Waku is best thought of as a single cohesive thing, there are three network interaction domains:
+While Waku v2 is best thought of as a single cohesive thing, there are three network interaction domains:
 
 (a) gossip domain
 (b) discovery domain
-(c) req/resp domain
+(c) request/reply domain
```

Comment:
I think we're trying to be consistent about calling this `request/response domain`